UI/Controls/MockEditControl.axaml.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using Avalonia.Controls;
+using Avalonia.Input;
+using Avalonia.Interactivity;
+using Avalonia.Markup.Xaml;
+using ReactiveUI;
+using UI.MessagesBus;
+
+namespace UI.Controls;
+
+internal partial class MockEditControl : UserControl
+{    
+    public static readonly RoutedEvent<RoutedEventArgs> UpdateRequestDataEvent =
+        RoutedEvent.Register<MockEditControl, RoutedEventArgs>(nameof(UpdateRequestData), RoutingStrategies.Bubble);
+    
+    public event EventHandler<RoutedEventArgs> UpdateRequestData
+    { 
+        add => AddHandler(UpdateRequestDataEvent, value);
+        remove => RemoveHandler(UpdateRequestDataEvent, value);
+    }
+    public MockEditControl()
+    {
+        InitializeComponent();
```

Comment:
Maybe turn on json highlighting for `MockEditor`? And disable hyperlinks/enable line highlight.